clients/unshaded/src/main/java/tachyon/client/TachyonFile.java

```diff
@@ -357,7 +368,7 @@ public boolean needPin() throws IOException {
    */
   public boolean promoteBlock(int blockIndex) throws IOException {
     FileBlockInfo blockInfo = getClientBlockInfo(blockIndex);
-    return mTachyonFS.promoteBlock(blockInfo.getBlockId());
+    return mTachyonFS.promoteBlock(blockInfo.blockInfo.getBlockId());
```

Comment:
how about change the name of blockInfo to fileBlockInfo? blockInfo.blockInfo doesn't seem to be intuitive.